### PR TITLE
User story #49

### LINKED ISSUE #49: [500] as an anonymous_user i want to search for projects
- [ ] i can research a project by names, ids, categories

### PR DESCRIPTION
**Pull request by galgros**
research bar from navbar working on project : title, description, zip code, project owner/ and on profils : fisrt/last name, id, zip code (redirection into home index)

### CODE REVIEW

File: src/View/Home/index.html.twig [hunk 1]
```diff
@@ -8,8 +8,18 @@
         <h1>Side by Side</h1>
         <a href="/Profil/all/">All Profils</a>
         <a href="/Project/new/"><button type="button" class="btn btn-warning">Lancez Votre Projet !</button></a>
-        {%  include '/Home/_profilsCarousel.html.twig' %}
-        {% include '/Home/_projectCard.html.twig' %}
+        <div>
+            <hr>
+            <h2>Projets</h2>
+            {% include '/Home/_projectCard.html.twig' %}
```
Comment (martinotm) [inline src/View/Home/index.html.twig]: Pas besoin du leading `/` devant le path du fichier à include.

Comment (galgros) [reply src/View/Home/index.html.twig]: Ok, corrigé

**Review by Cr7t3K — APPROVED** [review]
It's Good !

**Review by Raphonx — APPROVED** [review]
ok !

**Pull request merged**